README.md

```diff
@@ -361,6 +362,10 @@ open class ConsoleWriter: LogMessageWriter {
 }
 ```
 
+A default implementation is provided for the third `writeMessage` function in the `LogWriter`. The implementation 
+just calls the version of `writeMessage` that takes a `LogMessage` and `LogLevel`. By including this, it is not
+necessar to implement the new `writeMessage` method that takes a `LogMessageContext` in existing log writers.
```

Comment:
Is the "default implementation" referring to `func writeMessage(_ message: LogMessage, logLevel: LogLevel)`?
And "including this" is referring to the "third" function?